### PR TITLE
DOC-1959: Redial would, in some situations, cause select elements not to have an initial value selected when they should have.

### PR DESCRIPTION
**Pull request by kemister85**
Ticket: DOC-1959: Redial would, in some situations, cause select elements not to have an initial value selected when they should have.

Changes:
* added `DOC-1959: Redial would, in some situations, cause select elements not to have an initial value selected when they should have.` to staging.

Pre-checks:
- [x] Branch prefixed with `feature/6/` or `hotfix/6/`
- [x] Changelog entry added
- [x] ~`modules/ROOT/nav.adoc` has been updated (if applicable)~
- [x] ~Files has been included where required (if applicable~)
- [x] ~Files removed have been deleted, not just excluded from the build (if applicable)~
- [x] ~(New product features only) Release Note added~

Review:
- [x] Documentation Team Lead has reviewed

### CODE REVIEW

**Review by HAFRMO — APPROVED** [review]
I don't know if you want to add a note that this also solves a known bug in the Accessability plugin or not. It does, but it's probably not a must to mention it here.

**Comment by kemister85** [comment]
> I don't know if you want to add a note that this also solves a known bug in the Accessability plugin or not. It does, but it's probably not a must to mention it here.

@brian-forte Should we add a note as mentioned above?

File: modules/ROOT/pages/6.4.2-release-notes.adoc [hunk 1]
```diff
@@ -76,6 +76,13 @@ include::partial$misc/admon-releasenotes-for-stable.adoc[]
 
 // {productname} 6.4.2 also includes the following bug fixes:
 
+=== Redial would, in some situations, cause select elements not to have an initial value selected when they should have.
+
+In previous versions of {productname}, the browser did not like switching child elements within the accessibility checker dropdown menu.
+
+As a consequence, the dropdown menu would return without anything being selected by default in some situations.
+
+To fix this issue, if not explicitly set as empty by the user, {productname} will now populate the element unless user decides otherwise.
```
Comment (brian-forte) [inline modules/ROOT/pages/6.4.2-release-notes.adoc:85]:
```suggestion
=== Redial would, in some circumstances, cause elements to not have an initial value selected when they should have.

In previous versions of {productname}, the host browser did not properly manage switching child elements when the  the Accessibility Checker dialog was visible.

As a consequence, the dropdown menu in this dialog would, in some circumstances, present without one of the dropdown menu’s options being selected by default.

To fix this issue {productname} now populates the element with a default value, unless this menu is explicitly configured as empty by the particular {productname} instance’s configuration.
```

Edits and copy-edits.

File: modules/ROOT/pages/6.4.2-release-notes.adoc [hunk 2]
```diff
@@ -76,6 +76,13 @@ include::partial$misc/admon-releasenotes-for-stable.adoc[]
 
 // {productname} 6.4.2 also includes the following bug fixes:
 
+=== Redial would, in some circumstances, cause elements to not have an initial value selected when they should have.
+
+In previous versions of {productname}, the host browser did not properly manage switching child elements when the Accessibility Checker dialog was visible.
+
+As a consequence, the dropdown menu in this dialog would, in some circumstances, present without one of the dropdown menu’s options being selected by default.
+
+To fix this issue {productname} now populates the element with a default value, unless this menu is explicitly configured as empty by the particular {productname} instance’s configuration.
```
Comment (brian-forte) [inline modules/ROOT/pages/6.4.2-release-notes.adoc:85]:
```suggestion
To fix this issue {productname} now populates the element with a default value, unless this menu is explicitly configured as empty by the particular {productname} instance’s configuration.

NOTE: this change also addresses xref:6.4.1-release-notes.adoc#known-issues[a known issue] in the xref:a11ychecker.adoc[Accessibility Checker Premium plugin].
```

Conveniently enough, the Accessibility Checker known issue is the first item in the [‘Known Issues’ section of the _6.4.1 Release Notes_](https://tiny.cloud/docs/tinymce/6/6.4.1-release-notes/#known-issues). So, an xref to the `known-issues` section header links to said known issue, at least for practical purposes.

**Comment by brian-forte** [comment]
> Should we add a note as mentioned above?

See [my suggested edit](https://github.com/tinymce/tinymce-docs/pull/2686/files#r1183066033).

File: modules/ROOT/pages/6.4.1-release-notes.adoc [hunk 3]
```diff
@@ -1110,4 +1112,4 @@ After conducting a thorough investigation into the new **Advanced Template** plu
 
 We have identified that the "Insert link" dialog is an example of a dialog that does **not work** correctly in this regard, while the "Insert image" dialog functions as expected. These issues were discovered while examining mobile performance within the new **Advanced Template** plugin, and we are working to address them in order to provide a smoother and more user-friendly experience.
 
-Unfortunately, at this time there is no known workarounds for the issue with dialogs.
\ No newline at end of file
+Unfortunately, at this time there is no known workarounds for the issue with dialogs.
```
Comment (brian-forte) [inline modules/ROOT/pages/6.4.1-release-notes.adoc:1115]:
```suggestion
Unfortunately, at this time there is no known workaround for the issue with dialogs.
```
Typo correction.

**Review by brian-forte — APPROVED** [review]

**Pull request merged**